### PR TITLE
Pack merge tools to myget.org

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

File: src/nuspec/MergeDeveloperComments/MergeDeveloperComments.nuspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <id>MergeDeveloperComments</id>
+    <version>1.0.0</version>
+    <title>MergeDeveloperComments</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/dotnet/docfx/blob/dev/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/dotnet/docfx</projectUrl>
+    <description>This is a DocFX tool to merge developer comments.</description>
+    <copyright>Copyright 2015-2016</copyright>
```
Comment (superyyrrzz) [inline src/nuspec/MergeDeveloperComments/MergeDeveloperComments.nuspec]: >2016 [](start = 30, length = 4)

2017

File: src/nuspec/MergeSourceInfo/MergeSourceInfo.nuspec [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <id>MergeSourceInfo</id>
+    <version>1.0.0</version>
+    <title>MergeSourceInfo</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/dotnet/docfx/blob/dev/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/dotnet/docfx</projectUrl>
+    <description>This is a DocFX tool to merge source information.</description>
+    <copyright>Copyright 2015-2016</copyright>
```
Comment (superyyrrzz) [inline src/nuspec/MergeSourceInfo/MergeSourceInfo.nuspec]: >2016 [](start = 30, length = 4)

2017

File: src/nuspec/MergeSourceInfo/MergeSourceInfo.nuspec [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata>
+    <id>MergeSourceInfo</id>
```
Comment (hellosnow) [inline src/nuspec/MergeSourceInfo/MergeSourceInfo.nuspec]: >MergeSourceInfo [](start = 8, length = 15)

Need we prepend 'docfx' in the id?

**Comment by hellosnow** [comment]
:shipit:

File: src/nuspec/MergeSourceInfo/MergeSourceInfo.nuspec (hunk 3 above)
Comment (qinezh) [inline src/nuspec/MergeSourceInfo/MergeSourceInfo.nuspec]: no need currently, since it's only for internal usage.

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**